docs/multi_process.md

```diff
@@ -0,0 +1,127 @@
+# Using JAX in multi-host and multi-process environments
+
+(test-ref)=
+## Introduction
+
+This guide explains how to use JAX in environments such as Cloud TPU pods where
```

Comment:
Suggestion @skye : Maybe (but only maybe) - add a link for those users who aren't aware of what the pods are:

```suggestion
This guide explains how to use JAX in environments such as [Cloud TPU](https://cloud.google.com/tpu) pods where
```

Reply:
Great idea!